repos/system_upgrade/common/actors/removeupgradefailedflag/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import removeupgradefailedflag
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class RemoveUpgradeFailedFlag(Actor):
+    """
+    Removes upgrade failed flag file
+
+    The upgrade failed flag file is set in the initramfs when the upgrade fails
+    there to prevent rerunning the initramfs part of the upgrade.
+    It needs to be cleaned up before next leapp run.
+    """
+
+    name = 'remove_upgrade_failed_flag'
+    consumes = ()
+    produces = ()
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
```

Comment:
We are not supposed to touch the system during the `ChecksPhaseTag` (even if it was  created by us). Let's move this to the `InterimPreparationPhase`.